Rubberduck.Parsing/ParserRuleContextExtensions.cs

```diff
@@ -30,6 +30,12 @@ public static Selection GetSelection(this ParserRuleContext context)
                                  context.Stop.EndColumn() + 1);
         }
 
+        public static bool Contains(this ParserRuleContext context, ParserRuleContext otherContext)
```

Comment:
I would like to have some short doccomment on this to just allow IDE tooling to make clear that this is not an `Intersects` in usages, but that's honestly just a nitpick :)

Reply:
Sure, I will add that tomorrow.